src/stores/UserStore.js

```diff
@@ -0,0 +1,43 @@
+import { EventEmitter } from 'events';
+import Dispatcher from '../dispatcher/AppDispatcher';
+import ActionType from '../actions/ActionType';
+import usersData from '../data/users.json';
+
+const CHANGE_EVENT = 'CHANGE';
+let _users         = usersData;
+
+class UserStore extends EventEmitter {
+
+    constructor () {
+        super();
+
+        Dispatcher.register(this.registerActions.bind(this));
+    }
+
+    registerActions (action) {
+        switch (action.actionType) {
+            case ActionType.ADD_NEW_USER:
+                this._addNewUser(action.user);
```

Comment:
I think it's better to implement a class `User` and create an instance of this class to ensure that the added entries match those already in the list.